docs/config/build-options.md

```diff
@@ -53,8 +53,10 @@ Specify the directory to nest generated assets under (relative to `build.outDir`
 
 Imported or referenced assets that are smaller than this threshold will be inlined as base64 URLs to avoid extra http requests. Set to `0` to disable inlining altogether.
 
+Git LFS placeholders are automatically excluded from inlining because they do not contain the content of the file they represent.
+
 ::: tip Note
-If you specify `build.lib`, `build.assetsInlineLimit` will be ignored and assets will always be inlined, regardless of file size.
+If you specify `build.lib`, `build.assetsInlineLimit` will be ignored and assets will always be inlined, regardless of file size or being a Git LFS placeholder.
```

Comment:
Not for this PR, but we should find a way to avoid this in lib mode too.

Reply:
If I understand correctly, then there's a strong desire to inline everything in lib mode. Hence there are essentially three ways this can behave:

- Inline the LFS placeholder. Con: The resulting bundle is bricked. The LFS placeholder ins't the actual file content. It's how it behaves today.
- Skip inlining for the LFS placeholder. Con: Lib mode should bundle everything. So that would be an exception to that rule
- Display an error to the user. Con: It's an error

In regards to dealing with the situation in this PR there's these options:

- Keep current behavior and document it
- Keep current behavior but remove the highlighted sentence from the docs. Then the exact behavior for this edge case is in no longer documented and can be considered undefined behavior. Undefined behavior can be adjusted later without the need of a breaking change
- Pick another option from three possible behaviors above and document it

Reply:
Maybe we could keep the behaviour like in this PR, but if we encounter git lfs in lib build, we can warn about it to download the assets, as it's unlikely someone wants to build for a library that contains unreferenced assets (e.g. git lfs wouldn't work after npm publish)

Reply:
Alright. I'll see if I can make it produce a warning

Reply:
I pushed the code that emits a warning when encountering this edge case. I doubt that anyone will ever see this, though, because it seems rather unusual/unlikely that someone would use Git LFS in a library project :D